src/components/manage/Blocks/Search/SearchBlockView.jsx

```diff
@@ -106,4 +106,6 @@ export const SearchBlockViewComponent = compose(
   (Component) => React.memo(Component, blockPropsAreChanged),
 )(SearchBlockView);
 
-export default compose(withQueryString, withSearch())(SearchBlockViewComponent);
+export default compose(withSearch())(
+  compose(withQueryString)(SearchBlockViewComponent),
```

Comment:
I don't understand this diff. In effect, the execution order is the same, right?

Reply:
@tiberiuichim `compose` operates from right to left. So, before the change, withSearch() was applied first, but now it is applied second.

There's no need to use compose here; it could be written as withSearch()(withQueryString(SearchBlockViewComponent))

@ionlizarazu It would be helpful if you could explain a bit about how changing the order solves the problem.

Reply:
@tiberiuichim I found that this wasn't really the problem, but as @davisagli said, the order is different and it makes sense to me to execute first the withQueryString and after the withSearch.

I'll remove the composes to make it more readable.